src/app/components/NetworkRibbon/component/DetectionScreen.tsx

```diff
@@ -46,36 +55,42 @@ export function DetectionScreen(props: Props) {
           <img src={error_alert} alt="1" />
```

Comment:
This `alt` is a complete nonsense, please change it to something meaningful (e.g. 'icon error').